ingenialink/canopen/servo.py

```diff
@@ -31,7 +31,7 @@ class CanopenServo(Servo):
 
     MAX_WRITE_SIZE = CAN_MAX_WRITE_SIZE
 
-    STATUS_WORD_REGISTERS = "CIA402_DRV_STATE_STATUS"
+    STATUS_WORD_REGISTERS = "DRV_STATE_STATUS"
```

Comment:
This change should not affect CIA registers and manufacturer registers are equivalent:
https://drives.novantamotion.com/tsd/why-are-there-duplicated-registers-and-which-ones-#WhyarethereduplicatedregistersandwhichonesshouldIuse?-StatusWordConversionTable